src/Monolog/Handler/ZendMonitorHandler.php

```diff
@@ -57,22 +62,23 @@ public function __construct($level = Logger::DEBUG, $bubble = true)
     protected function write(array $record)
     {
         $this->writeZendMonitorCustomEvent(
-            $this->levelMap[$record['level']],
+            Logger::getLevelName($record['level']),
             $record['message'],
-            $record['formatted']
+            $record['formatted'],
+            $this->levelMap[$record['level']]
         );
     }
 
     /**
-     * Write a record to Zend Monitor
-     *
-     * @param int    $level
-     * @param string $message
-     * @param array  $formatted
+     * Write to Zend Monitor Events
+     * @param string $type Text displayed in "Class Name (custom)" field
+     * @param string $message Text displayed in "Error String"
+     * @param mixed $formatted Displayed in Custom Variables tab
+     * @param int $severity Set the event severity level (-1,0,1)
      */
-    protected function writeZendMonitorCustomEvent($level, $message, $formatted)
```

Comment:
:( this function is protected..  adding an extra argument might be a BC for some devs.

Reply:
Yes.. but it also appears like this code never worked, so I am thinking probably nobody uses this or if they do it's breaking and they didn't notice. I doubt extending code is something to worry about.